app/assets/stylesheets/component_guide/application.scss

```diff
@@ -1,6 +1,43 @@
 // This file contains the styles for the Component Guide.
 
-@import "govuk_publishing_components/all_components";
+// This is the file that the application needs to include in order to use
+// the components.
+
+// feature flag for accessible link styles
+$govuk-new-link-styles: true;
+
+@import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/component_support";
+
+// Include all govuk frontend components
+@import "govuk/components/all";
```

Comment:
Why do we need to include all govuk-frontend components? I tried removing this line and couldn't see any immediate problems, but that's not to say there weren't any.

Reply:
Good spot, I included this as it is [currently included in `all_components`](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/_all_components.scss#L10-L11), after testing this change locally, I don't think this is required, will do some further testing and see if Percy picks up on any visual changes.

Reply:
After removing this import, [Percy shows a change in the `label` component](https://percy.io/55ba5e1a/govuk_publishing_components/builds/26892110/changed/1496462152?browser=firefox&browser_ids=39&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=original&width=1280&widths=375%2C1280), with the radio input circle no longer being displayed.

There appears to be an existing rendering issue in the component guide for [label component](https://components.publishing.service.gov.uk/component-guide/label/preview):

> Note that this example will not render correctly - do not be alarmed. It only works properly when inside the radio component.

I can at least keep the radio input circle by also ensuring the radio stylesheet is imported into application.css